src/cljs/main/broadfcui/utils.cljs

```diff
@@ -103,22 +103,12 @@
     "local.broadinstitute.org"
     (string/join "." (rest (string/split js/window.location.hostname ".")))))
 
-(defn get-leo-cookie-domain []
-  (if (= "local.broadinstitute.org" js/window.location.hostname)
-    "broadinstitute.org"
-    (string/join "." (rest (string/split js/window.location.hostname ".")))))
-
 (defn delete-access-token-cookie []
   (.remove goog.net.cookies "FCtoken" "/" (get-cookie-domain)))
 
 (defn set-access-token-cookie [token]
   (if token
-    (.set goog.net.cookies "FCtoken" token -1 "/" (get-cookie-domain))
-    (delete-access-token-cookie)))
-
-(defn set-notebooks-access-token-cookie [token]
-  (if token
-    (.set goog.net.cookies "FCtoken" token -1 "/" (get-leo-cookie-domain))
+    (.set goog.net.cookies "FCtoken" token -1 "/" (get-cookie-domain) true) ; secure cookie
```

Comment:
`goog.net.cookies` supports the secure flag:
https://google.github.io/closure-library/api/goog.net.Cookies.html

Reply:
are you gonna switch to `LeoToken` in some later PR?

Reply:
This is actually the Orch token (`LeoToken` is set by the server). Figured I might as well make this secure too.

Reply:
I wish Clojure had Java-style comments so you could do `(.set ... /* secure */ true)`

Reply:
Yeah I’ve always liked that pattern :)

Reply:
You could ignore a symbol or string:

```clojure
(.set ... #_secure true)
```

Reply:
That's a good point! I guess I'd still wish it colored the expression as a comment in my editor. In any case, the need for this suggests that the option might make more sense as a keyword argument or a map.